### PR TITLE
added no-fossil-fuels-no-co2 haiku

### PR DESCRIPTION
**Pull request by fabestah**

### CODE REVIEW

File: _haikus/2022_no-fossil-fuels-no-co2.md
Comment (ltagliaferri) [inline _haikus/2022_no-fossil-fuels-no-co2.md:8]: This line has one syllable too many (assuming C-O-2), I suggest this:

```suggestion
bust fossil fuels, CO2, <br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request. The changes requested have not been addressed over the course of a few days, so we are closing this PR due to inactivity and are marking it as `invalid`. If you would like to try again, please open a new pull request.

**Comment by fabestah** [comment]
Sorry for the inactivity, I wasn't able to get online the last few days!

The Haiku Syllable Counter actually stated "bust fossil fuels, no CO2," are exactly 7 syllables but I will open a new pull request with your suggested version.

Thank you very much for doing this btw, it really helps to get the hang of GitHub 😊❤️

**Pull request closed without merging**
